sentry/src/main/java/io/sentry/SentryLogEvent.java

```diff
@@ -0,0 +1,182 @@
+package io.sentry;
+
+import static io.sentry.DateUtils.doubleToBigDecimal;
+
+import io.sentry.protocol.SentryId;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryLogEvent implements JsonUnknown, JsonSerializable {
+
+  private @NotNull SentryId traceId;
+  private @NotNull Double timestamp;
+
+  private @NotNull String body;
```

Comment:
haven't seen it in the docs, but maybe you know - do we need to do any truncation here? E.g. if there's a requirement that relay will drop logs with body longer than XXX or something

Reply:
The same limit that applies to error envelopes also applies here. Not truncation needed / wanted. We probably want to implement a weight based approach in the future to only send a lower batch size in some cases.

Reply:
And the limit applies to the whole batch.